build_tools/tests/fileset_tool_test.py

```diff
@@ -14,18 +14,31 @@
 FILESET_TOOL = Path(__file__).parent.parent / "fileset_tool.py"
 
 ARTIFACT_DESCRIPTOR_1 = r"""
+[components.doc]
+#extends = "lib"
```

Comment:
Is it intended to have this commented out?

Reply:
Thanks.